app/web/src/components/AssetDetailsPanel.vue

```diff
@@ -146,7 +146,7 @@
           @focus="focus"
         />
       </Stack>
-      <template v-if="ffStore.SHOW_INTRINSIC_EDITING">
+      <template>
```

Comment:
@wendybujalski @jobelenus I believe blank template is useless here, right?

Reply:
Well... that answers that.
<img width="990" alt="Screenshot 2024-09-24 at 5 04 22 PM" src="https://github.com/user-attachments/assets/c4371a38-ba9a-45bf-a8e9-edd5380ae59b">